repo2docker/buildpacks/repo2docker-entrypoint

```diff
@@ -6,7 +6,7 @@ if [[ ! -z "${R2D_ENTRYPOINT:-}" ]]; then
     if [[ ! -x "$R2D_ENTRYPOINT" ]]; then
         chmod u+x "$R2D_ENTRYPOINT"
     fi
-    exec "$R2D_ENTRYPOINT" "$@"
+    exec "$R2D_ENTRYPOINT" "$@" 2>&1 | tee /jupyter-server-log.txt
```

Comment:
Won't this require the entrypoint to be run as `root` in order to write to `/`?

Reply:
Good question. I hadn't thought that far yet, was still at the "does this even work" stage :D

Reply:
You were right and our testing robots found it as well :D